R/pkg/R/DataFrame.R

```diff
@@ -2873,6 +2873,7 @@ setMethod("intersect",
 #' @rdname except
 #' @export
 #' @note except since 1.4.0
+#' @note behaviour changed from \code{EXCEPT ALL} to \code{EXCEPT DISTINCT} in 2.0.
```

Comment:
Ur.. I think we have use `@note` for version specification in SparkR. Just adding

```
Note: blabla
```
should be fine like other places.

Reply:
I don't mind it either way, but to note:
- r doc order and whitespace is significant, if you use `#' Note:` you must put it after L2856, if you put an extra `#'` ie. empty line that it becomes the `Details` section, which might be the right place; see http://spark.apache.org/docs/latest/api/R/awaitTermination.html

Reply:
ie.
```
#' but not in another SparkDataFrame. This is equivalent to \code{EXCEPT DISTINCT} in SQL.
#'
#' Note: Before Spark 2.0.0, the behavior was equivalent to `EXCEPT ALL` in SQL.
#'
#' @param x a SparkDataFrame.
```

Reply:
Nit: This is wrong.